### ISSUE TITLE
[Azure Bot Service] Error when registering a bot

### ISSUE THREAD

**Issue by ohmusama** [Microsoft/BotBuilder#1435]
I get the error: **This item cannot be accessed or modified** when trying to create a new bot.

![capture](https://cloud.githubusercontent.com/assets/220289/19399002/2eb8f720-9204-11e6-9d1c-9cb9442c4e09.PNG)

**Comment by danmarshall** [comment]
Thanks for reporting this. Meanwhile, can you close your browser and try again?

**Comment by ohmusama** [comment]
I switched to chrome, worked in chrome.

**Comment by danmarshall** [comment]
If you can give any details on the browser & version that would be helpful. Thanks!

**Comment by ohmusama** [comment]
Microsoft Edge 38.14393.0.0
Microsoft EdgeHTML 14.14393

**Comment by chrimc62** [comment]
@danmarshall is this something we can fix or have to live with?

**Comment by danmarshall** [comment]
Closing this issue for now, please feel free to open a new issue if you encounter it.

**Comment by ajit-drag** [comment]
Also coming in chrome for me.
Version 56.0.2924.87 (64-bit)

**Comment by hailiu2586** [comment]
if you can try sign out of https://dev.botframework.com and sign back in, it might clear up your browser side state and unblock you. We will investigating for a code fix in the mean time.

**Comment by ajit-drag** [comment]
I tried open in incognito, it solved my problem but the app id and app secret were gone. So I have to create new app id and secret. Other wise opening in incognito worked for me.

**Comment by EricDahlvang** [comment]
I'm closing this issue, as it is no longer occurring.

**Comment by IRRDC** [comment]
The issue is occuring in IE 11 (11.0.9600.18697) for me. Used the Chrome workaround with Version 59.0.3071.115 (64-Bit).

**Comment by GuilhermeQuelhos** [comment]
I'm facing the same issue on IE 11 (11.0.9600.18697).
I tried the workaround using Chrome (Version 59.0.3071.115 (Official Build) (64-bit) ) but it didn't work for me. 
Using Chrome, I cannot select any template.
![image](https://user-images.githubusercontent.com/7399322/27908225-b6b562c6-6242-11e7-9d9c-b3e6688fde91.png)

**Comment by danmarshall** [comment]
@IRRDC @GuilhermeQuelhos - thanks for your comments. Are you seeing the exact same issue as in the screenshot at the top of this issue? If not, can you open a new issue since this may not actually be the same situation.

**Comment by IRRDC** [comment]
I see the same message as in the original screenshot when I try to create a bot based on the first 3 templates. Also when I try to edit (in IE) the bot I created using Chrome I end up not on the bot's page but on the template selection page.

**Comment by anishetty** [comment]
1) Same issue is occurring on IE 11 for first three templates. 
2) When Language Understanding is chosen it ask to sign in and after successful sign in it gives option to create new blank LUIS app or use LUIS api key to connect. When clicked on continue it again displays the same message.

**Comment by nwhitmont** [comment]
Open a new issue if you are still having problems
